### PR TITLE
Pure pursuit

### PR DESCRIPTION
**Pull request by RahulBansal1112**

### CODE REVIEW

File: src/main/java/frc/robot/trajectory/PurePursuit.java [hunk 1]
```diff
@@ -124,4 +124,13 @@ public Translation2d findLookahead(Translation2d robotPos, double robotHeading)
 			? pathPoints.get(lookaheadPointIndex) : pathPoints.get(pathPoints.size() - 1);
 		return lookaheadPoint;
 	}
+
+	/**
+	 * asdf.
```
Comment (AG6GR) [inline src/main/java/frc/robot/trajectory/PurePursuit.java]: ?

**Review by AG6GR — CHANGES_REQUESTED** [review]
Please remember to push branches for #15 and #16 before merging this.

**Comment by RahulBansal1112** [comment]
> 

We cleaned up the drive code (there's some small commented out pieces of code, but we're still using those to test the auto paths). As for the teleop states, we've written the code for it and tested it, but need to test one part of it to make sure it 100% works before we'll merge it into main.

File: src/main/java/frc/robot/trajectory/PurePursuit.java [hunk 2]
```diff
@@ -12,8 +12,8 @@
 	private ArrayList<Translation2d> keyPoints = new ArrayList<>();
 	private ArrayList<Translation2d> pathPoints = new ArrayList<>();
 	private int lastClosestPointIndex = 0;
-	// lookahead point is this many inches ahead
-	private final int lookaheadDistance = 6;
+	// lookahead point is this many points ahead
+	private final int lookaheadDistance = 8;
```
Comment (AG6GR) [inline src/main/java/frc/robot/trajectory/PurePursuit.java]: Constants should be static and all caps

File: src/main/java/frc/robot/Robot.java [hunk 3]
```diff
@@ -35,7 +35,7 @@
 	private GrabberFSM grabberSystem;
 
 	// Constants
-	private static final boolean RUN_COMPRESSOR = true;
+	private static final boolean RUN_COMPRESSOR = false;
```
Comment (AG6GR) [inline src/main/java/frc/robot/Robot.java]: Keep this true on main branch. In fact maybe we should put this on shuffleboard.

**Review by AG6GR — CHANGES_REQUESTED** [review]

**Review by AG6GR — APPROVED** [review]

**Pull request merged**